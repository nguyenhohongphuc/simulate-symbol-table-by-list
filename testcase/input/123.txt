INSERT x number
INSERT y string
BEGIN
INSERT z number
ASSIGN x 123
LOOKUP x
LOOKUP z
BEGIN
INSERT y string
LOOKUP y
END
LOOKUP y
END