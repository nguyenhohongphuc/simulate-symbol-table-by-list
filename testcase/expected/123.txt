success
success
success
success
0
1
success
2
0